src/libraries/System.Runtime.Experimental/ref/System.Runtime.Experimental.csproj

```diff
@@ -19,6 +19,8 @@
     <IsPackable>true</IsPackable>
     <PackageDescription>Exposes new experimental APIs from System.Runtime</PackageDescription>
     <PackageId>$(MSBuildProjectName)</PackageId>
+    <!-- TODO: Remove when the package shipped with NET6. -->
```

Comment:
I believe this package will be removed in .NET 7 anyways. cc: @tannergooding

Reply:
Yes, assuming it doesn't get delayed by something else

Reply:
Let's keep the comment as-is until we know for sure.